lib/screen/detail/artwork_detail_page.dart

```diff
@@ -550,6 +544,42 @@ class _ArtworkDetailPageState extends State<ArtworkDetailPage>
             },
           ),
         ],
+        OptionItem(
+          title: 'view_on_'.tr(args: [asset.secondaryMarketName]),
+          icon: SvgPicture.asset(
+            'assets/images/external_link.svg',
+            width: 20,
+            height: 20,
+          ),
+          onTap: () {
+            unawaited(
```

Comment:
Please check if the link is valid. If the link is not valid, disable this option.